### PR TITLE
Allow Identity Learning Rate Scheduler

### PR DESCRIPTION
**Pull request by kmdalton**
The control flow in `pytorch_fob/optimizers/lr_schedulers/__init__.py` was preventing me from setting the `scheduler` to "identity". I think this change restores it to the intended pattern, but maybe I'm missing something with how this is supposed to work.

### CODE REVIEW

**Comment by kmdalton** [comment]
Oops sorry - this needs to be rebased. Let me start over.
